clients/python/moondream/__init__.py

```diff
@@ -1,5 +1,13 @@
 from .types import VLM
 from .onnx_vl import OnnxVL
+from .cloud_vl import CloudVL
+
+
+def VL(model_path: str = None, api_key: str = None) -> VLM:
```

Comment:
Looked it up, turns out it's possible to force people to use named parameters.

```python
def foo(*, arg0="default0", arg1="default1", arg2="default2"):
    pass
```

should we do that?